epregressions/tests/test_runtests.py

```diff
@@ -1004,7 +1004,10 @@ def test_base_case_but_multi_process(self):
         diff_results = r.run_test_suite()
         # there should be 1 file result
         self.assertEqual(2, len(diff_results.entries_by_file))
-        results_for_file = diff_results.entries_by_file[0]
+        if diff_results.entries_by_file[0].basename == 'my_file':
+            results_for_file = diff_results.entries_by_file[0]
+        elif diff_results.entries_by_file[1].basename == 'my_file':
+            results_for_file = diff_results.entries_by_file[1]
```

Comment:
This was an interesting one, previously the two files would always come out in the serial order, but now sometimes they would come out backwards.  I had to simply check which entry was the one I'm inspecting before calling assertions on it.